netket/operator/_fermion2nd/numba.py

```diff
@@ -146,7 +147,15 @@ def _flattened_kernel(  # pragma: no cover
         term_split_idxs,
         cutoff,
         pad=False,
+        constraint=False,
     ):
+        def hash_raw(x):
+            """Numba can not hash arrays, so we hash them manually"""
+            r = 0
+            for u in x:
+                r = r * 7 + u
```

Comment:
? is this safe

Reply:
It is not, but the worst would be not removing due to collision and than getting an error.